chapters/object_types/volumes.txt

```diff
@@ -20,28 +20,23 @@ ANARIVolume anariNewVolume(ANARIDevice, const char *subtype);
 Extension: `KHR_VOLUME_TRANSFER_FUNCTION1D`
 
 The 1D transfer function volume is created by passing the subtype string
-`transferFunction1D` to `anariNewVolume`. It supports the folowing parameter:
+`transferFunction1D` to `anariNewVolume`. It supports the folowing parameters:
 
 .Parameters understood by the transferFunction1D volume.
 [cols="<3,<3,>2,<9",options="header,unbreakable"]
 |===================================================================================================
 | Name             | Type                 | Default | Description
 | field            |`SPATIAL_FIELD`       |         | <<object_types_spatial_field, Spatial field>> used for the field values of the volume
 | valueRange       |`FLOAT32_BOX1` / `FLOAT64_BOX1` |  [0, 1] | sampled values of `field` are clamped to this range
-| color            |`FLOAT32_VEC3` / `FLOAT32_VEC4` / `ARRAY1D` of <<Color>>|         | array to map sampled and clamped field values to color
-| opacity          |`FLOAT32` / `ARRAY1D` of `FLOAT32`|         | opacity value multiplied into alpha component of the `color` parameter
+| color            |`FLOAT32_VEC4` / `FLOAT32_VEC3` / `ARRAY1D` of <<Color>>| (1.0, 1.0, 1.0, 1.0)  | array to map sampled and clamped field values to color
+| opacity          |`FLOAT32` / `ARRAY1D` of `FLOAT32`|        1.0 | opacity value multiplied into alpha component of the `color` parameter
+| attenuationDistance |`FLOAT32` |        1.0 | attenuation distance after which the transmittance of the volume turns into opacity
```

Comment:
this will need some discussion: attenuation is usually absorption plus out-scattering (and is often colored / wavelength-dependent), whereas we see opacity as monochromatic absorption or rather density (because it also controls the emission)